.circleci/config.yml

```diff
@@ -3,75 +3,75 @@ orbs:
   codecov: codecov/codecov@3.2.4
   sonarcloud: sonarsource/sonarcloud@1.1.1
 
+executors:
+  linux-docker:
+    docker:
+      - image: cimg/base:2023.03
+
 jobs:
```

Comment:
Its a great news that the CircleCI build issue is finally fixed 👍
Does adding `linux-docker` as `executors` fix the problem?

Reply:
Thank you, it is indeed great news that CircleCI finally fixed macOS executors with our accounts.
> Does adding linux-docker as executors fix the problem?

Adding `linux-docker` as `executors` helps to reuse the executor for `sonar_analysis` and `codecov_upload` jobs (so the short answer would be no).

The executor for `build-and-test` which requires macOS remains `macos.x86.medium.gen2`, but the problem with running macOS executors is fixed externally by CircleCI. 🙂